src/test/scala/chiseltest/tests/RegionsTest.scala

```diff
@@ -18,35 +18,42 @@ class RegionsTest extends AnyFlatSpec with ChiselScalatestTester {
         c.clock.step()
         c.in.poke(70.U)
         c.clock.step()
-      }.fork.withRegion(Monitor) {
-        c.in.expect(42.U)
-        c.clock.step()
-        c.in.expect(70.U)
-      }.joinAndStep(c.clock)
+      }.fork
+        .withRegion(Monitor) {
+          c.in.expect(42.U)
+          c.clock.step()
+          c.in.expect(70.U)
+        }
+        .joinAndStep(c.clock)
     }
   }
 
   it should "resolve read-after-write dependencies, even if threads in opposite order" in {
     test(new PassthroughModule(UInt(8.W))) { c =>
-      fork.withRegion(Monitor) {
-        c.in.expect(42.U)
-        c.clock.step()
-        c.in.expect(70.U)
-      }.fork {
-        c.in.poke(42.U)
-        c.clock.step()
-        c.in.poke(70.U)
-        c.clock.step()
-      }.joinAndStep(c.clock)
+      fork
+        .withRegion(Monitor) {
+          c.in.expect(42.U)
+          c.clock.step()
+          c.in.expect(70.U)
+        }
+        .fork {
```

Comment:
Highly undesirable... and also inconsistent with the above, where it put the fork on the close curly braces?